test/e2e/network/kube_proxy.go

```diff
@@ -127,6 +128,8 @@ var _ = SIGDescribe("Network", func() {
 			},
 		}
 		fr.PodClient().CreateSync(hostExecPod)
+		defer fr.PodClient().DeleteSync(hostExecPod.Name, &metav1.DeleteOptions{}, framework.DefaultPodDeletionTimeout)
```

Comment:
@spiffxp I think that this was the problem,  the exec pod was executing with pause and executes a command in a goroutine https://github.com/kubernetes/kubernetes/pull/86924/files#diff-e99584717e21a274f44c98f0c97a3182L232
This way it has to be cleaned